plugins/caddy/Caddyfile

```diff
@@ -3,31 +3,30 @@
     order souin_cache before rewrite
     souin_cache {
         headers Content-Type Authorization
-        olric {
-            url `olric:3320`
-        }
+        log_level info
         ttl 1000
     }
 }
 
 :80
 
-souin_cache {
-    route /testing-purpose {
-        headers Content-Type
-    }
-    route /testing-purpose-2 {
-        headers Content-Type
-        ttl 100
-    }
-    route /a* {
-        ttl 20
-    }
-    route /a* {
-        headers Content-Type
-    }
-    route * {
-        headers Authorization
-        ttl 666
-    }
+@match path /test1*
+@match2 path /test2*
+@matchdefault path /default
+@matchall path /*
+
+souin_cache @match {
+    ttl 30s
+    headers Cookie
 }
+
+souin_cache @match2 {
+    ttl 50s
+    headers Authorization
+}
+
+souin_cache @matchdefault {
+    ttl 25s
+}
+
+souin_cache @matchall {}
```

Comment:
This isn't valid syntax, you can't have a closing brace on the same line as an opening brace. Best to just remove them.

Also, just omit the matcher altogether, or use `*` to match all requests. Using `path /*` will create a path matcher that makes a string comparison, which is ever so slightly less efficient than just not having a path matcher, which avoids that string comparison.

```suggestion
souin_cache
```